common/middleware/set-location-items.js

```diff
@@ -13,14 +13,35 @@ function setLocationItems(locationType, fieldName) {
     }
 
     try {
-      const locations = await referenceDataService.getLocationsByType(
-        locationType
+      if (!Array.isArray(locationType)) {
+        locationType = [locationType]
+      }
+
+      const locationString = locationType[0]
+
+      const locations = (
+        await Promise.all(
+          locationType.map(x => referenceDataService.getLocationsByType(x))
+        )
       )
+        .reduce((acc, val) => acc.concat(val))
+        .sort((a, b) => {
```

Comment:
This may have already been thought of and decided against, but, could we have the `location_type` filter in the API support multiple values? Similar to what we have done with other filters like location IDs...?

Then we wouldn't need to really do a lot of this multiple call + sort heavy lifting?

Reply:
Very good point. Will raise on Monday

Reply:
FE LGTM. The location type filter is not currently in scope. I will put this as an improvement for a future sprint.